src/components/Ability.js

```diff
@@ -1,26 +1,24 @@
 import PropTypes from 'prop-types';
 
-const Ability = ({ability}) => {
-  const descriptions = ability.desc.map((desc, index) => <p key={index} >{desc}</p>);
+const Ability = ({ page }) => {
+  const descriptions = page.desc.map((desc, index) => <p key={index} >{desc}</p>);
 
   return (
-    <main>
-      <h2 className='main__heading'>{ability.full_name} ({ability.name})</h2>
-      <div className='main__divider'></div>
+    <>
       {descriptions}
-    </main>
+    </>
   )  
 }
```

Comment:
`Ability` and `Skill` look like the exact same component. can you turn them into just one?

Reply:
After I configure the Loading component, I'm going to focus on refactoring all directories into a single component 👍